src/DropDown/DropDown.component.tsx

```diff
@@ -140,12 +146,13 @@ export class DropDown extends React.Component<DropDownProps> {
             fromEvent(this.rootElement, 'touchstart')
         ).pipe(
             filter(() => this.state.clicked),
-            filter(({ target }) => !dropDown.contains(target))
+            filter(({ target }) => !dropDown.contains(target)),
+            throttleTime(500)
         );
         const dropDownClick$ = merge(
             fromEvent(dropDown, 'click'),
             fromEvent(dropDown, 'touchstart')
-        );
+        ).pipe(throttleTime(500));
```

Comment:
Mobile devices fire both click and touchstart events, so the dropdown was being "clicked" and therefore toggled twice. I'm just throttling it so that it only fires 1 event within 500ms

Reply:
Worth making the throttle time a prop so later tweaking (if necessary) won't require another release?

Reply:
Almost everything I read online said there was ~300-350ms delay between the events. I went for 500 to be safe.

Leaning against prop because if I'm wrong about my assumptions, I'll need to fix the defaults here anyway. Prop also a bit weird because the throttle times are set in the componentDidMount, so it wouldn't respond to prop value changes over the life of the component.

Reply:
I'd be less concerned about the prop changing as much as having a way to allow configuration of the throttle. Not a hill to die on though.